src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,31 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, chosenTabId, setChosenTabId, activeTab }) => {
```

Comment:
The `Tabs` component should use `activeTabId` and `onTabSelected` props instead of `chosenTabId` and `setChosenTabId`. Update these props to align with the task requirements.